code/modules/mob/living/basic/heretic/fire_shark.dm

```diff
@@ -31,7 +31,10 @@
 
 /mob/living/basic/fire_shark/Initialize(mapload)
 	. = ..()
-	AddElement(/datum/element/death_drops, list(/obj/effect/gibspawner/human))
+	var/static/list/death_loot
+	if(!death_loot)
+		death_loot = string_list(list(/obj/effect/gibspawner/human))
```

Comment:
Out of curiosity, why do you normally create some static list and use string lists for others? 

Like the one above is `/obj/item/stack/rods` which I can't imagine is any less common than `/obj/effect/gibspawner/human` (Unless I'm completely wrong)

Reply:
> Out of curiosity, why do you normally create some static lists and use string lists for others?

This is because some elements have lists that, despite being static, are identical to lists also used in other instances of the same element. For example: some syndicate mobs spawn gibs on death, much like the fire shark. And if you look closely, you'll see this is not the only case.